UnityProject/Assets/Scripts/Player/PlayerSync.cs

```diff
@@ -473,10 +473,6 @@ private void UpdateMe()
 
 	private void Synchronize()
 	{
-		if (isLocalPlayer && GameData.IsHeadlessServer)
```

Comment:
Technically this can trigger on the headless server,m idk how it handles its own playerr

Reply:
I'm under the impression that headless is server only? There shouldn't be a localplayer for something that is not a client.

Reply:
Yeah, i dont think this check was ever true. So removing it wouldnt change anything. But just making it a Headless check could break stuff, so would need testing.

Reply:
![image](https://user-images.githubusercontent.com/56727168/120243227-f64eb600-c25e-11eb-85a3-74612d2a66aa.png)

i donno maybe we should keep it in. Might be some weird interaction with headless